sdk/core/core-http/package.json

```
@@ -44,8 +44,6 @@
     "es/src/**/*.js.map",
     "types/src/**/*.d.ts",
     "types/src/**/*.d.ts.map",
```

I think this needs to be updated as well to
```suggestion
    "types/**/*.d.ts",
    "types/**/*.d.ts.map",
```
Since now there is a types/latest and types/3.1 folder

---

yes, just fixed.